backend/tracim_backend/lib/search/elasticsearch_search/es_models.py

```diff
@@ -129,6 +144,7 @@ class IndexedContent(Document):
     # information about content are stored in the "file_data" fields not defined
     # in this mapping
     b64_file = Text()
+    file_data = Object(FileData)
```

Comment:
Warning !
`FILE_PIPELINE_SOURCE_FIELD` should match `file_data` here to work as expected.

Reply:
What should I do here?

Reply:
don't really know. @grignards make `FILE_PIPELINE_SOURCE_FIELD` configurable for some reason that i don't really get now.